### DISCUSSION TITLE
How to setup dev environment on Mac OS.

### DISCUSSION THREAD

**Discussion by spleenware** [InfiniTimeOrg/InfiniTime#1787] [Q&A]
I'd like to have a go at enhancements and new apps for the PineTime, but feel a bit lost on how to get VSCode setup on Mac OS.
The documentation seems a bit sparse on how to get setup.

**Comment by VeryUsual** [comment]
https://code.visualstudio.com/docs/setup/mac

**Comment by JF002** [comment]
The documentation only focuses on Linux since most of the developers are using this OS.
I honestly don't know Mac OS, but basically, you need a MacOS verson of the toolchain ([which ARM seems to provide here](https://developer.arm.com/downloads/-/gnu-rm)) and CMake. If you can get them to work, you should be able to follow the build instructions [here](https://github.com/InfiniTimeOrg/InfiniTime/blob/main/doc/buildAndProgram.md). 

There's also a [docker container](https://github.com/InfiniTimeOrg/InfiniTime/blob/main/doc/buildWithDocker.md) that can be used to build the project, but I don't know if it works on MacOS.
